corehq/apps/app_manager/templates/app_manager/form_view.html

```diff
@@ -166,6 +165,38 @@
     <div class="tab-content appmanager-tab-content">
       {% if form.uses_cases %}
         <div class="tab-pane" id="case-configuration">
+          {% if case_property_warning.show %}
+            <div class="alert alert-warning">
+              <button
+                id="performance-warning-toggle"
+                type="button"
+                class="close"
```

Comment:
This doesn't sit right with me. Are you giving it class = "close" just to have it positioned in the right top corner? But this is not actually a close action. I feel this is hack but if we already do it in other places I won't be too strict on it.

Reply:
Yeah it positions it in the top right corner. I didn't know of another straightforward way to do it, but agree it is a bit of a hack.